### DISCUSSION TITLE
pitch issues

### DISCUSSION THREAD

**Discussion by andrewsha123** [flybywiresim/aircraft#2376] [Help]
Hi, my a3nx keeps having a glitch even if im 5 percent pitch down my vertical speed is 0. I am on the latest build and my CG is normal and the weather conditions are clear.

**Comment by MapleNAV-TCASFAULT** [comment]
What flight model are you using? If you're using the legacy model, it'll behave like that.

**Reply by MapleNAV-TCASFAULT** [reply]
also are you using the CM a321? And are you using the dev or stable? If you're using the stable try out the dev version!
